includes/Server.hpp

```diff
@@ -67,36 +67,35 @@ enum Command {
 
 class Server {
  private:
-  static bool                _signal;
-  int                        _socketFd;
-  int                        _port;
-  std::string                _startTime;
-  std::string                _password;
-  clientsMap                 _clients;
-  struct sockaddr_in         _address;
+  static bool _signal;
+  int _socketFd;
+  int _port;
+  std::string _startTime;
+  std::string _password;
+  clientsMap _clients;
+  struct sockaddr_in _address;
   std::vector<struct pollfd> _pollFds;
-  channelsMap                _channels;
+  channelsMap _channels;
 
  public:
-  explicit Server(int port, std::string password);
-
+  explicit Server(int port, const std::string &password);
+#ifdef TEST
+  Server() {};
+#endif
   /* Getters */
-  int getSocketFd(void) const;
-  int getPort(void) const;
-  const std::string &getPassword(void) const;
-  const Client &getClientByFd(int fd) const;
-  Channel &getChannelByName(const std::string &name);
+  int getSocketFd() const;
+  int getPort() const;
+  const std::string &getPassword() const;
+  Client &getClientByFd(int fd);
+  const Channel &getChannelByName(const std::string &name) const;
+  const channelsMap &getChannels() const;
+  const clientsMap &getClients() const;
 
-  /* Server Mounting */
-  void runServer(void);
-  void createSocket(void);
-  void createPoll(void);
-  void fetchStartTime(void);
-  void monitorConnections();
-  static void signalHandler(int signal);
+  /* Setters */
+  void setStartTime(void);
```

Comment:
This function `void setStartTime(void)` has been deleted in a previous merge. Please delete it from the header file